{{cookiecutter.role_name}}/LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2019 EPAM Systems
 
```

Comment:
should be replaced for using variables value from cookiecutter

Reply:
Done